lib/resty/etcd/v3.lua

```diff
@@ -647,6 +647,7 @@ local function request_chunk(self, endpoint, method, scheme, host, port, path, o
             if health_check.conf ~= nil then
                 health_check.report_failure(endpoint.http_host)
             end
+            return nil, body.error.http_status .. ": " .. endpoint.http_host
```

Comment:
Should be  endpoint.http_host .. ": " .. body.error.http_status？
Can you add a test for it?

Reply:
Working on it

Reply:
Format changed thx~

Since the health check test mocks the situation when one endpoint is failed before but not during interaction with etcd, the error would occur at the time creating watch connection but not during watch. If we want to cover the last situation, we need to kill etcd in the middle of the test, which seems a bit complicated for the test but it's still feasible. Do I need to implement that?